### PR TITLE
Add customizable messages to paidtimeout module

### PR DESCRIPTION
**Pull request by ALazyMeme**
Pull request checklist:

- [ ] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable
- [x] I have tested all changes

### CODE REVIEW

**Comment by ALazyMeme** [comment]
I need to make the commands page dynamic. Obviously it shouldn't show anything if the messages are blank. What is the best way of going about this? Seems to be the last thing I need in order to complete the pr.

File: pajbot/modules/paidtimeout.py [hunk 1]
```diff
@@ -175,7 +245,7 @@ def load_commands(self, **options):
                     CommandExample(
                         None,
                         f"Timeout someone for {self.settings['timeout_length2']} seconds",
-                        chat=f"user:!{self.settings['command_name2']} paja\nbot>user: You just used {self.settings['cost2']} points to time out paja for an additional {self.settings['timeout_length2']} seconds.",
+                        chat=f"user:!{self.settings['command_name2']} karl_kons\nbot>user: {self.settings['message_to_additional_timeouter'].format(victim='karl_kons', cost=self.settings['cost'], duration=self.settings['timeout_length'])}",
```
Comment (pajlada) [inline pajbot/modules/paidtimeout.py:248]: use `get_phrase` instead of running `.format` on the setting

File: pajbot/modules/paidtimeout.py [hunk 2]
```diff
@@ -160,7 +230,7 @@ def load_commands(self, **options):
                 CommandExample(
                     None,
                     f"Timeout someone for {self.settings['timeout_length']} seconds",
-                    chat=f"user:!{self.settings['command_name']} paja\nbot>user: You just used {self.settings['cost']} points to time out paja for an additional {self.settings['timeout_length']} seconds.",
+                    chat=f"user:!{self.settings['command_name']} karl_kons\nbot>user: {self.settings['message_to_additional_timeouter'].format(victim='karl_kons', cost=self.settings['cost'], duration=self.settings['timeout_length'])}",
```
Comment (pajlada) [inline pajbot/modules/paidtimeout.py:233]: use `get_phrase` instead of running `.format` on the setting

File: pajbot/modules/paidtimeout.py [hunk 3]
```diff
@@ -101,36 +137,70 @@ def base_paid_timeout(self, bot, source, message, _time, _cost):
...
+                    bot.send_message(
+                        source,
+                        self.settings["message_to_timeouter"].format(victim=victim, cost=_cost, duration=_time),
+                        method="whisper",
+                    )
+
+                if self.settings["message_to_victim"] != "":
+                    bot.send_message(
+                        victim,
+                        self.settings["message_to_victim"].format(source=source, duration=_time),
```
Comment (pajlada) [inline pajbot/modules/paidtimeout.py:200]: use `get_phrase` instead of running `.format` on the setting

File: pajbot/modules/paidtimeout.py [hunk 4]
```diff
@@ -101,36 +137,70 @@ def base_paid_timeout(self, bot, source, message, _time, _cost):
...
             else:
-                bot.whisper(source, f"You just used {_cost} points to time out {victim} for {_time} seconds.")
-                bot.whisper(
-                    victim,
-                    f"{source} just timed you out for {_time} seconds LUL",
-                )
+                if self.settings["message_to_timeouter"] != "":
+                    bot.send_message(
+                        source,
+                        self.settings["message_to_timeouter"].format(victim=victim, cost=_cost, duration=_time),
```
Comment (pajlada) [inline pajbot/modules/paidtimeout.py:193]: use `get_phrase` instead of running `.format` on the setting

File: pajbot/modules/paidtimeout.py [hunk 5]
```diff
@@ -101,36 +137,70 @@ def base_paid_timeout(self, bot, source, message, _time, _cost):
...
+                    bot.send_message(
+                        victim,
+                        self.settings["additional_message_to_victim"].format(source=source, duration=_time),
+                        method="whisper",
+                    )
+
+                if self.settings["message_to_additional_timeouter"] != "":
+                    bot.send_message(
+                        source,
+                        self.settings["message_to_additional_timeouter"].format(
```
Comment (pajlada) [inline pajbot/modules/paidtimeout.py:181]: use `get_phrase` instead of running `.format` on the setting

File: pajbot/modules/paidtimeout.py [hunk 6]
```diff
@@ -101,36 +137,70 @@ def base_paid_timeout(self, bot, source, message, _time, _cost):
...
                 victim.timeout_end += datetime.timedelta(seconds=_time)
-                bot.whisper(victim, f"{victim}, you were timed out for an additional {_time} seconds by {source}")
-                bot.whisper(
-                    source, f"You just used {_cost} points to time out {victim} for an additional {_time} seconds."
-                )
+
+                if self.settings["additional_message_to_victim"] != "":
+                    bot.send_message(
+                        victim,
+                        self.settings["additional_message_to_victim"].format(source=source, duration=_time),
```
Comment (pajlada) [inline pajbot/modules/paidtimeout.py:174]: use `get_phrase` instead of running `.format` on the setting

**Review by pajlada — CHANGES_REQUESTED** [review]

**Comment by ALazyMeme** [comment]
@pajlada Maybe I'm missing something, but the example you provided on stream doesn't seem to sufficiently cover the fact that there are 2 commands. Should I split it into example1 and example2 variables? Keep in mind there there are different timeout values and costs (1 for each command).

image

**Pull request closed without merging**